packages/cli/__tests__/daemon/__integration__/disable/__scripts__/daemon_disable.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -e
+
+echo "================ daemon enable ==============="
```

Comment:
A little picky, but this disables the daemon.

Reply:
yep.